src/core/sorted_map.h

```diff
@@ -78,9 +79,8 @@ class SortedMap {
     return std::visit(Overload{[](const auto& impl) { return impl.MallocSize(); }}, impl_);
   }
 
-  // TODO: to get rid of this method.
-  dict* GetDict() const {
-    return std::get<RdImpl>(impl_).dict;
+  uint64_t Scan(uint64_t cursor, absl::FunctionRef<void(std::string_view, double)> cb) const {
+    return std::visit(Overload{[&](const auto& impl) { return impl.Scan(cursor, cb); }}, impl_);
```

Comment:
Stop using the overload 🤣

Reply:
@dranikpg Why? it's a nice utility  ? :D

Reply:
Because it works without in all those cases, its an auto lambda and `visit([](auto& impl) { ... })` does the job. I pointed it out already in the last PR, but the code was already written 🙂